src/lib/Components/Bidding/Elements/Typography.tsx

```diff
@@ -44,23 +44,23 @@ const SerifItalic = styled.Text`
   ${maxWidth}
 `
 
-export const Sans12 = props => <Sans fontSize={1} {...props} />
-export const Sans14 = props => <Sans fontSize={2} {...props} />
-export const Sans16 = props => <Sans fontSize={3} {...props} />
-export const Sans18 = props => <Sans fontSize={4} {...props} />
+export const Sans12 = props => <Sans fontSize={1} lineHeight={1} {...props} />
+export const Sans14 = props => <Sans fontSize={2} lineHeight={5} {...props} />
+export const Sans16 = props => <Sans fontSize={3} lineHeight={6} {...props} />
+export const Sans18 = props => <Sans fontSize={4} lineHeight={8} {...props} />
 
-export const Serif12 = props => <Serif fontSize={1} {...props} />
-export const Serif14 = props => <Serif fontSize={2} {...props} />
-export const Serif16 = props => <Serif fontSize={3} {...props} />
-export const Serif18 = props => <Serif fontSize={4} {...props} />
+export const Serif12 = props => <Serif fontSize={1} lineHeight={1} {...props} />
+export const Serif14 = props => <Serif fontSize={2} lineHeight={2} {...props} />
+export const Serif16 = props => <Serif fontSize={3} lineHeight={3} {...props} />
```

Comment:
This should be `lineHeight={5}` (`== lineHeight: 26px`)

Reply:
👍Thanks for catching these!

Reply:
I now understand why you had chosen `3` in the first place. It seems that in the zeplin doc `20px` is specified so `3` (`== 20px`) seemed to make more sense. We probably should implement and start using typography components with a `t` suffix.